src/cc/bpf_module.cc

```diff
@@ -349,10 +348,13 @@ int BPFModule::annotate() {
       fn->addFnAttr(Attribute::AlwaysInline);
 
   // separate module to hold the reader functions
-  auto m = make_unique<Module>("sscanf", *ctx_);
+  auto m = std::make_unique<Module>("sscanf", *ctx_);
 
   size_t id = 0;
-  for (auto &table : *tables_) {
+  Path path({std::to_string((uintptr_t)this)});
+  for (auto it = ts_->lower_bound(path), up = ts_->upper_bound(path); it != up; ++it) {
+    TableDesc &table = it->second;
```

Comment:
For now we need it to provide a stable ordering for the users of the index-based API. If we phase that out, then sure.

My motivation for this changeset is in fact to allow the TableStorage class to be used externally to the JIT-time objects (BPFModule). Having a long-lived BPFFS that gets passed in and owned separately is one of the next steps.